pkgs/applications/science/chemistry/d-seams/default.nix

```diff
@@ -0,0 +1,32 @@
+{ clangStdenv, stdenv, fetchurl, catch2, rang, fmt, libyamlcpp, cmake, eigen
+, lua, luaPackages, liblapack, blas, lib, boost, gsl ? null }:
+
+clangStdenv.mkDerivation rec {
+  version = "v1.0.1";
+  pname = "d-SEAMS";
+
+  src = fetchurl {
```

Comment:
Can we use 'fetchFromGitHub' here instead?

Reply:
Sure, have switched to it now. Good catch!